src/components/DashboardWorkPoolFlowRunsTotal.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <span class="dashboard-work-pool-flow-runs-total">{{ allRunsCount }}</span>
+</template>
+
+<script lang="ts" setup>
+  import { useSubscription } from '@prefecthq/vue-compositions'
+  import { computed } from 'vue'
+  import { useWorkspaceApi } from '@/compositions'
+  import { WorkPool, FlowRunsFilter } from '@/models'
+  import { mapper } from '@/services'
+  import { WorkspaceDashboardFilter } from '@/types'
+
+  const props = defineProps<{
+    workPool: WorkPool,
+    filter?: WorkspaceDashboardFilter,
+  }>()
+
+  const subscriptionOptions = {
+    interval: 30000,
+  }
```

Comment:
Can we use the composition you wrote? Or is that not ready to use

Reply:
I used that composition on the dashboard specific components, but left off these child components for now, since they could be used outside the dashboard. We talked about working out a composition inheritance type of utility, I figured we could tackle polling on this page more holistically in another PR. Open to pushback on that though.